modules/ossm-installing-multi-primary-network-mesh.adoc

```diff
@@ -0,0 +1,290 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-installing-multi-primary-multi-network-mesh_{context}"]
+= Installing a multi-primary multi-network mesh 
+:context: ossm-installing-multi-primary-multi-network-mesh
+
+Install {istio} in the multi-primary multi-network configuration on two {ocp-product-title} clusters. 
+
+[NOTE]
+====
+In this procedure, `CLUSTER1` is the East cluster and `CLUSTER2` is the West cluster. 
+====
+
+You can adapt these instructions for a mesh spanning more than two clusters.
+
+.Prerequisites
+
+* You have installed the {SMProduct} Operator on all of the clusters that comprise the mesh.
+
+* You have completed _Creating certificates for a multi-cluster mesh_. 
+
+* You have completed _Applying certificates to a multi-cluster topology_.
+
+* You have `istioctl` installed on the laptop you will use to run these instructions.
+
```

Comment:
You have created an IstioCNI resource.

Add one more, pre-req

Reply:
done